### PR TITLE
fix(Auth): sign in with google

### PR DESCRIPTION
**Pull request by ruiguoamz**
*Description of changes:*
This PR is a short term fix for [documentation issue](https://github.com/aws-amplify/docs/issues/3068) (`Social Sign In With Google`)

Long term fix would be to link to social provider managed resource instead of homebrew for every providers (Facebook, Google, Apple, Amazon, etc)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/auth/fragments/common/social_signin_web_ui/setup_auth_provider.md [hunk 1]
```diff
@@ -21,15 +21,26 @@
...
 ![Image](~/images/cognitoHostedUI/google5.png)
-3. Create your OAuth2.0 credentials by choosing *OAuth client ID* from the *Create credentials* drop-down list.
-![Image](~/images/cognitoHostedUI/google6.png).
-4. Choose *Web application*.
-5. Click *Create*.
-6. Note the *OAuth client ID* and *client secret*. You will need them for the next section in the CLI flow.
-7. Choose *OK*.
+7. Click *CREATE*
+![Image](~/images/cognitoHostedUI/google6.png)
+8. Type in *App Information* and *Developer contact information* which are required field and click *SAVE AND CONTINUE* three times to finish setting up consent screen
```
Comment (thisisabhash) [inline docs/lib/auth/fragments/common/social_signin_web_ui/setup_auth_provider.md:37]: Suggestion for wording : Click Save and Continue in the next steps (OAuth consent screen -- Scopes -- Test Users)

**Comment by thisisabhash** [comment]
docs/images/cognitoHostedUI/google8.png - Suggestion to remove personal information from top part of this image and crop half of the image for better readability of text.
Screen Shot 2021-05-14 at 1 52 18 PM

**Review by thisisabhash — APPROVED** [review]
Added few minor comments. LGTM

**Review by div5yesh — APPROVED** [review]
LGTM 🌮  on behalf of JS and Android.

**Review by changxu0306 — APPROVED** [review]
lgtm

**Pull request merged**